metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -762,8 +809,9 @@ def _populate_projects_registry(self) -> None:
 
     def _authenticate(self) -> None:
         try:
-            self.server = self.config.make_tableau_client()
-            logger.info("Authenticated to Tableau server")
+            site_content_url = self.current_site.content_url if self.current_site else self.config.site
```

Comment:
This is done because I saw there is some code that does re-authentication: https://github.com/datahub-project/datahub/blob/3dc5326e440b2479b06fed9362da0f4c7713b635/metadata-ingestion/src/datahub/ingestion/source/tableau.py#L814. With this we always re-authenticate with the current site or fallback to the site property provided in the config.